pkg/apis/experimental/v1alpha1/types_swagger_doc_generated.go

```diff
@@ -356,7 +356,8 @@ func (JobStatus) SwaggerDoc() map[string]string {
 }
 
 var map_NodeUtilization = map[string]string{
-	"": "NodeUtilization describes what percentage of a particular resource is used on a node.",
+	"":      "NodeUtilization describes what percentage of a particular resource is used on a node.",
+	"value": "The accepted values are from 0 to 1.",
```

Comment:
Why did this get updated? Is head broken, or does our presubmit check not include experimental?